domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/service/display/DisplayGroupAdapterService.java

```diff
@@ -30,12 +30,17 @@
     private DisplayGroupRepository displayGroupRepository;
```

Comment:
DisplayGroupAdapterService is growing too much. Long methods. Can we do some heavy refactorings please? Introduce multiple self explanatory private methods for example

Reply:
done